opennms-base-assembly/pom.xml

```diff
@@ -708,6 +708,11 @@
       <groupId>org.jsoup</groupId>
       <artifactId>jsoup</artifactId>
     </dependency>
+    <dependency>
```

Comment:
You are mentioning about jsoup dependency or protocols? I used collector class 'org.opennms.protocols.xml.collector.XmlCollector' in collectd and this class is present in this particular package.